crates/typst-library/src/foundations/context.rs

```diff
@@ -61,7 +61,7 @@ fn require<T>(val: Option<T>) -> HintedStrResult<T> {
 }
 
 /// Executes a `context` block.
-#[elem(Construct, Introspectable)]
+#[elem(Construct, Unqueriable, Introspectable)]
```

Comment:
on second thought, could you revert the unqueriable on `ContextElem` and `LayoutElem`? I realized that I'm not sure on the exact semantics of `Unqueriable` (specifically, whether you should still be able to do in `context query(here())` to fetch the context block after all). As this will break at the very least the try/catch snippet from the Typst examples book, this would come up there as potential alternative and I don't want to break that twice.